### PR TITLE
Ingredients editor

### PR DESCRIPTION
**Pull request by ccomb**
Many fixes and ergonomic improvements for the ingredient/process editor

### CODE REVIEW

File: data/food/export_agb/activities.json [hunk 1]
```diff
@@ -946,39 +938,40 @@
...
     "density": 0.6195,
     "inedible_part": 0.1,
     "transport_cooling": "none",
     "visible": true,
+    "bvi": 0.0,
     "complements": {
-      "agro-diversity": 0,
-      "agro-ecology": 0
+      "agro-diversity": 10,
+      "agro-ecology": 10
```
[the remark this answers is not in the document]
Comment (ccomb) [reply data/food/export_agb/activities.json:954]: thx, I'm asking Lisa

[the remark this answers is not in the document]
Comment (ccomb) [reply data/food/export_agb/activities.json:954]: Which one is the other ?  If this is the animal-welfare of the eggplant, it's part of an automatic intergrity cleanup of the data

[the remark this answers is not in the document]
Comment (magopian) [reply data/food/export_agb/activities.json:954]: Yep: https://github.com/MTES-MCT/ecobalyse/pull/326/files/da713333bb589cc0760d4c3c50b0650b637dc2e8#diff-2d06c5cf11819ecaae3989af591d97c352898c5a3db9ba07c04e3e3b91980536L979-R974

File: data/notebooks/ingredients.py [hunk 2]
```diff
@@ -49,25 +89,29 @@ def reverse(d):
...
     "subingredient_organic": "Sous-ingrédient bio",
-    "ratio": "Ratio Complexe/Simple",
-    "complements.agro-diversity": "Agro Diversity",
-    "complements.agro-ecology": "Agro Ecology",
-    "complements.animal-welfare": "Animal Welfare",
+    "ratio": "Ratio",
+    # complements
+    "complements.agro-diversity": "AgroDiv",
+    "complements.agro-ecology": "AgroEco",
+    "complements.animal-welfare": "Animal welf",
```
[the remark this answers is not in the document]
Comment (ccomb) [reply data/notebooks/ingredients.py:114]: The dotted notation is used by [flatdict](https://flatdict.readthedocs.io/en/stable/) to reconstruct the dict

**Pull request merged**